src/ImageSharp/Image.FromStream.cs

```diff
@@ -770,25 +770,11 @@ public static Image Load(Configuration configuration, Stream stream, out IImageF
                 throw new NotSupportedException("Cannot read from the stream.");
             }
 
-            // To make sure we don't trigger anything with aspnetcore then we just need to make sure we are
-            // seekable and we make the copy using CopyToAsync if the stream is seekable then we aren't using
-            // one of the aspnetcore wrapped streams that error on sync api calls and we can use it without
-            // having to further wrap
-            if (stream.CanSeek)
-            {
-                if (configuration.ReadOrigin == ReadOrigin.Begin)
```

Comment:
So here we're resetting the input stream position before we attempt to decode because [`Configuration.ReadOrigin`](https://docs.sixlabors.com/api/ImageSharp/SixLabors.ImageSharp.Configuration.html#SixLabors_ImageSharp_Configuration_ReadOrigin) has been set. We need to ensure that we still respect that configuration option.

Reply:
Ahhh now I get what you were saying.  Somehow I completely missed that bit.  I'll add that back in.